tests/unit_tests/visualization/test_legend_handlers.py

```diff
@@ -0,0 +1,46 @@
+from simtools.util import names
+from simtools.visualization import legend_handlers as leg_h
+
+
+def test_objects():
+    object_list = [
+        leg_h.EdgePixelObject,
+        leg_h.LSTObject,
+        leg_h.MSTObject,
+        leg_h.MeanRadiusOuterEdgeObject,
+        leg_h.OffPixelObject,
+        leg_h.PixelObject,
+        leg_h.SCTObject,
+        leg_h.SSTObject,
+    ]
+
+    for obj in object_list:
+        instance = obj()
+        assert isinstance(instance, object)
```

Comment:
I guess this test is more for the sake of coverage? Otherwise it doesn't really do much.

Reply:
Yes, indeed. I wanted to test the object but I didn't see another way, so at least testing if it exists is something we could do.